services/movie_session.py

```diff
@@ -0,0 +1,53 @@
+from datetime import datetime
+
+from django.db.models import QuerySet
+
+from db.models import MovieSession, Movie, CinemaHall
+
+
+def create_movie_session(
+        movie_show_time: datetime,
+        movie_id: int,
+        cinema_hall_id: int,
+) -> MovieSession:
+
+    movie = Movie.objects.get(id=movie_id)
+    cinema_hall = CinemaHall.objects.get(id=cinema_hall_id)
+
+    movie_session = movie.movie_sessions.create(
```

Comment:
Use `MovieSession` model to create instance. Besides that, avoid using lines 14 and 15, write ids directly while creating movie session instance